### ISSUE TITLE
[Bug]: mdb & mdb_* ratings return always the same value

### ISSUE THREAD

**Issue by xiaodoudou** [Kometa-Team/Kometa#1023]
### Version Number

1.17.3-develop12

### What branch are you on?

nightly

### Describe the Bug

Hi,

Since several weeks (would 2 to 3), the data returned by mdb & mdb_* on the rating are always the same.

For example:
- `mdb_trakt` will always return `7.4` regardless of which item (meaning all the library items will have the same score)
- `mdb_tmdb` will always return `8.1` regardless of which item (meaning all the library items will have the same score)
- `mdb_imdb`  will always return `7.1` regardless of which item (meaning all the library items will have the same score)

Kind of clueless what's going on.

Let me know if I'm missing something.

### Relevant Collection/Overlay/Playlist Definition

```yml
TV Shows:
    operations:
      delete_collections_with_less: 1
      delete_unmanaged_collections: true
      mass_audience_rating_update: mdb_trakt
      mass_user_rating_update: mdb_tmdb
      mass_critic_rating_update: mdb_imdb
    overlay_path:
    - remove_overlays: false
    - file: /config/audience_rating_left_trakttv.yml
    - file: /config/user_rating_center_tmdb.yml
    - file: /config/critic_rating_right_imdb.yml
```

### Logs

https://gist.github.com/xiaodoudou/864bc607f8301e7e8b39d3958fd48604

**Comment by YozoraXCII** [comment]
This is a known issue in 1.17.3 - first time it's been posted on GH though.

You can revert to 1.17.2 to resolve this until a bugfix is pushed.

**Comment by meisnate12** [comment]
this is fixed in 1.18.0
